src/lib/recoil/jobs.js

```diff
@@ -0,0 +1,90 @@
+import {atom, noWait, selector, selectorFamily} from "recoil";
+import {httpState, optionsState} from "./base";
+import {careerState, careersParamsState} from "./career";
+
+export const careerJobsState = atom({key: "career-modal-jobs", default: {fetching: false, jobs: []}});
+export const jobOptionsState = atom({key: "job-options", default: {}});
```

Comment:
If we switch to passing the options back directly to the widget, I think we can remove this in favor of just using something like: `get(optionsState)?.career?.jobs || {}`

Reply:
jobOptions are now passed in through props!